test/github.ts

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const nock = require('nock');
+import * as nock from 'nock';
```

Comment:
I love that this requires literally the opposite change as `chalk` ... modules, literally the worst.